<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="7" passed="6">
  <reporter-output>
    <line>
      <![CDATA[[2017_08_01_15_49_09_login] ,截图失败，原因：null]]>
    </line>
    <line>
      <![CDATA[<img src="../../[2017_08_01_15_49_09_login] ,截图失败，原因：null"/>]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="14802" started-at="2017-08-01T07:48:56Z" finished-at="2017-08-01T07:49:11Z">
    <groups>
    </groups>
    <test name="Login" duration-ms="14802" started-at="2017-08-01T07:48:56Z" finished-at="2017-08-01T07:49:11Z">
      <class name="com.blueway.test.uit.testcase.login.Login_001">
        <test-method status="PASS" signature="startTest(org.testng.ITestContext)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_001@1cecbb6e]" name="startTest" is-config="true" duration-ms="5136" started-at="2017-08-01T15:48:56Z" finished-at="2017-08-01T15:49:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@101dd23e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method status="PASS" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_001@1cecbb6e]" name="login" duration-ms="513" started-at="2017-08-01T15:49:01Z" data-provider="testData" finished-at="2017-08-01T15:49:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=success, USERNAME=admin932, PASSWORD=8888, CODE=0000}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.blueway.test.uit.testcase.login.Login_001@1cecbb6e]" name="endTest" is-config="true" duration-ms="1285" started-at="2017-08-01T15:49:02Z" finished-at="2017-08-01T15:49:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.blueway.test.uit.testcase.login.Login_001 -->
      <class name="com.blueway.test.uit.testcase.login.Login_002">
        <test-method status="PASS" signature="startTest(org.testng.ITestContext)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="startTest" is-config="true" duration-ms="4052" started-at="2017-08-01T15:49:03Z" finished-at="2017-08-01T15:49:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@101dd23e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method status="PASS" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="login" duration-ms="433" started-at="2017-08-01T15:49:07Z" data-provider="testData" finished-at="2017-08-01T15:49:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=此用户不存在!, USERNAME=, CASENAME=用户名为空, PASSWORD=8888, CASEID=T001, CODE=0000}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="login" duration-ms="393" started-at="2017-08-01T15:49:08Z" data-provider="testData" finished-at="2017-08-01T15:49:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=此用户不存在!, USERNAME=abcd001, CASENAME=用户名不存在, PASSWORD=8888, CASEID=T002, CODE=0000}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="login" duration-ms="379" started-at="2017-08-01T15:49:08Z" data-provider="testData" finished-at="2017-08-01T15:49:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=密码错误!, USERNAME=10011001, CASENAME=密码为空, PASSWORD=, CASEID=T003, CODE=0000}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="login" duration-ms="386" started-at="2017-08-01T15:49:09Z" data-provider="testData" finished-at="2017-08-01T15:49:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=此用户不存在!, USERNAME=10011001, CASENAME=密码错误, PASSWORD=88888, CASEID=T004, CODE=0000}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[期望的文字是 [此用户不存在!] 但是找到了 [密码错误!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 期望的文字是 [此用户不存在!] 但是找到了 [密码错误!]
	at org.testng.Assert.fail(Assert.java:94)
	at com.blueway.test.uit.utils.SeleniumUtils.isTextCorrect(SeleniumUtils.java:120)
	at com.blueway.test.uit.pages.impl.LoginPage.checkUserName(LoginPage.java:36)
	at com.blueway.test.uit.testcase.login.Login_002.login(Login_002.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[[2017_08_01_15_49_09_login] ,截图失败，原因：null]]>
            </line>
            <line>
              <![CDATA[<img src="../../[2017_08_01_15_49_09_login] ,截图失败，原因：null"/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="login" duration-ms="350" started-at="2017-08-01T15:49:09Z" data-provider="testData" finished-at="2017-08-01T15:49:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=*验证码错误!!, USERNAME=10011001, CASENAME=密码码为空, PASSWORD=8888, CASEID=T005, CODE=}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.util.Map)[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="login" duration-ms="364" started-at="2017-08-01T15:49:09Z" data-provider="testData" finished-at="2017-08-01T15:49:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{EXP_RESULT=*验证码错误!!, USERNAME=10011001, CASENAME=密码码错误, PASSWORD=8888, CASEID=T005, CODE=00000}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.blueway.test.uit.testcase.login.Login_002@76dff374]" name="endTest" is-config="true" duration-ms="1185" started-at="2017-08-01T15:49:10Z" finished-at="2017-08-01T15:49:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.blueway.test.uit.testcase.login.Login_002 -->
    </test> <!-- Login -->
  </suite> <!-- Suite -->
</testng-results>
